File: src/java/eu/sig/training/ch05/boardpanel/v2/Rectangle.java
package eu.sig.training.ch05.boardpanel.v2;

import java.awt.Point;

// tag::Rectangle[]
public class Rectangle {
    private final Point position;
    private final int width;
    private final int height;

    public Rectangle(Point position, int width, int height) {
        this.position = position;
        this.width = width;
        this.height = height;
    }

    public Point getPosition() {
        return position;
    }

    public int getWidth() {
        return width;
    }

    public int getHeight() {
        return height;
    }
}
// end::Rectangle[]

File: src/csharp/eu/sig/training/ch05/boardpanel/v2/Rectangle.cs
using System.Drawing;

namespace eu.sig.training.ch05.boardpanel.v2
{
    // tag::Rectangle[]
    public class Rectangle
    {
        public Point Position { get; set; }

        public int Width { get; set; }

        public int Height { get; set; }

        public Rectangle(Point position, int width, int height)
        {
            this.Position = position;
            this.Width = width;
            this.Height = height;
        }

    }
    // end::Rectangle[]
}